### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by bjiro**
Grammar fixes

### CODE REVIEW

**Review by dvershinin — CHANGES_REQUESTED** [review]
Please don't introduces more grammmar problems 🤔

File: readme.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ if __name__ == '__main__':
 	)
 ```
 
-InstaDM create table `message` if not exists.
+InstaDM createes a table `message` if it does not already exists.
```
Comment (CamTosh) [inline readme.md:38]:
```suggestion
InstaDM creates a table `message` if it does not already exists.
```

**Comment by CamTosh** [comment]
@dvershinin can you resolve conflicts?

**Comment by dvershinin** [comment]
I could, but "Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests". 

The changes to readme are so minor, I'd think just manually repeating them on master is easier than anything.

**Pull request closed without merging**